src/cart/crt_init.c

```diff
@@ -23,13 +23,15 @@ dump_envariables(void)
 {
 	int	i;
 	char	*val;
-	char	*envars[] = {"CRT_PHY_ADDR_STR", "D_LOG_STDERR_IN_LOG",
+	char	*envars[] = {"D_PROVIDER", "D_INTERFACE", "D_DOMAIN", "D_PORT",
```

Comment:
Why do we have more env vars now ? they seem to be duplicate of previous OFI_PORT etc ?

Reply:
@johannlombardi wanted to standardize envariables as names like CRT_PHY_ADDR_STR are more confusing